src/Microsoft.Win32.Registry.AccessControl/ref/Microsoft.Win32.Registry.AccessControl.cs

```diff
@@ -5,6 +5,10 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Security.AccessControl.RegistryAccessRule))]
```

Comment:
Is there any way to tell the tool to keep these type forwards?

Reply:
Go look at the source in Arcade.  Type forwards are a little tricky: should the ref have a typeforward or a typedef?  That depends on wether type was pushed down or type is only meant to live in a different runtime assembly.  You should think about how you want to tool it.